src/templates/playgroundTemplate.ts

```diff
@@ -2,39 +2,42 @@ const template = `// MongoDB Playground
 // To disable this template go to Settings | MongoDB | Use Default Template For Playground.
 // Make sure you are connected to enable completions and to be able to run a playground.
 // Use Ctrl+Space inside a snippet or a string literal to trigger completions.
+// The result of the last command run in a playground is shown on the results panel.
+// By default the first 20 documents will be returned with a cursor.
+// Use 'console.log()' to print to the debug output.
 
 // Select the database to use.
 use('mongodbVSCodePlaygroundDB');
 
-// The drop() command destroys all data from a collection.
-// Make sure you run it against the correct database and collection.
-db.sales.drop();
-
 // Insert a few documents into the sales collection.
-db.sales.insertMany([
-  { '_id': 1, 'item': 'abc', 'price': 10, 'quantity': 2, 'date': new Date('2014-03-01T08:00:00Z') },
-  { '_id': 2, 'item': 'jkl', 'price': 20, 'quantity': 1, 'date': new Date('2014-03-01T09:00:00Z') },
-  { '_id': 3, 'item': 'xyz', 'price': 5, 'quantity': 10, 'date': new Date('2014-03-15T09:00:00Z') },
-  { '_id': 4, 'item': 'xyz', 'price': 5, 'quantity':  20, 'date': new Date('2014-04-04T11:21:39.736Z') },
-  { '_id': 5, 'item': 'abc', 'price': 10, 'quantity': 10, 'date': new Date('2014-04-04T21:23:13.331Z') },
-  { '_id': 6, 'item': 'def', 'price': 7.5, 'quantity': 5, 'date': new Date('2015-06-04T05:08:13Z') },
-  { '_id': 7, 'item': 'def', 'price': 7.5, 'quantity': 10, 'date': new Date('2015-09-10T08:43:00Z') },
-  { '_id': 8, 'item': 'abc', 'price': 10, 'quantity': 5, 'date': new Date('2016-02-06T20:20:13Z') },
+db['sales'].insertMany([
```

Comment:
If we want “proper” programmatic usage, this is probably the right way to go, since it handles leading underscores well + could potentially be used with TS (if there were public TS definitions for mongosh):

```suggestion
db.getCollection('sales').insertMany([
```

Reply:
What is the reason for this change?

Reply:
From Slack:

> Changed how we use collection methods collection methods from `db.sales.find` to `db['sales'].find` to reduce possible rough spots with folks that have spaces and dashes in their collection names.

Reply:
I like Anna's suggestion!

Reply:
Just added it to the description, sorry only had it in the slack.
One thing to note is that `db.getCollection('...')` doesn't suggest collection names currently. It won't be too hard to add though. I'll create a ticket to add it once we decide if we add it here.